utils/subscription.ts

```diff
@@ -11,3 +15,25 @@ export const serviceOwnerCheckTask = (
     (svcId: NonEmptyString) => svcId === ownerSubscriptionId,
     _ => ResponseErrorForbiddenNotAuthorized
   )(serviceId);
+
+/**
+ * This check verifies that:
+ *  - isVisible === false || (isVisible === true && svcMetadata !== undefined)
+ * @param isVisible
+ * @param serviceMetadata
+ * @returns serviceMetadata if the check is successful; otherwise ResponseErrorValidation.
+ *
+ */
+export const serviceVisibleMetadataCheckTask = (
```

Comment:
1) function that return booleans and variables which are booleans should be prefix with `is` or `has`
(es. `isValidServiceTask`)

2) not required for this PR, but a safer way to make these is to build a io-ts type, somenthing like:

```ts
const ValidServicePayload = t.intersection([ ServicePayload, 
 t.union([
     t.interface({ is_visible: t.literal(false) }) 
     t.interface({ is_visible: t.literal(true), services_metadata: ServiceMetadata })
  ]) 
]);
```

then in your code this task become:

```ts
fromEither(ValidServicePayload.decode(...))
   .chain(updateService)
```

and updateService / createService take `ValidServicePayload` as input.

In this way you're _forced_ to call `ValidServicePayload.decode` before any update or create operation
(while actually they may work with invalid service payloads).